src/main/java/io/vlingo/common/PentaConsumer.java

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2012-2018 Vaughn Vernon. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+
+package io.vlingo.common;
+
+import java.util.Objects;
+import java.util.function.Consumer;
+/**
+ * Represents an operation that accepts five input arguments and returns no
+ * result.  This is the five-arity specialization of {@link Consumer}.
+ * Unlike most other functional interfaces, {@code PentaConsumer} is expected
+ * to operate via side-effects.
+ *
+ * <p>This is a <a href="package-summary.html">functional interface</a>
+ * whose functional method is {@link #accept(Object, Object, Object, Object, Object)}.
+ *
+ * @param <T> the type of the first argument to the operation
+ * @param <U> the type of the second argument to the operation
+ * @param <V> the type of the third argument to the operation
+ * @param <X> the type of the fourth argument to the operation
+ * @param <X> the type of the fifth argument to the operation
```

Comment:
Should be `<Y>`. Looks like you faked out Javadocs since it usually catches missing generic parameters.